pages/implementation/reference/typescript.mdx

```diff
@@ -150,7 +150,7 @@ Custom destination interface format:
 };
 ```
 
-In Node each callback gets an additional first parameter - `userId` and, optionally, a `anonymousId` parameter. To get the `anonymousId` parameter in your workspace [reach out to us](https://www.avo.app/docs/help/troubleshooting).
+In Node each callback is asynchronous and gets an additional first parameter - `userId` and, optionally, a `anonymousId` parameter. To get the `anonymousId` parameter in your workspace [reach out to us](https://www.avo.app/docs/help/troubleshooting).
```

Comment:
Should we add something more about this here, or link to a doc on userIds and anonIds in server side tracking (if we don't have one already, we should)?

Reply:
yes, definitely

Reply:
We don't have it now, so I'll do in a separate pr